### DISCUSSION TITLE
Dumb aware

### DISCUSSION THREAD

**Discussion by aarcangeli** [dsoftwareinc/ghactions-manager#63] [Q&A] [answered]
Hi, is there a particular reason why the tool window is not available during indexing?

![image](https://user-images.githubusercontent.com/538137/201528050-53e7395c-7b91-4052-8c9f-7e045d497942.png)

**Comment by cunla** [comment, marked as answer]
Good point.
I am still new to jetbrains plugins - how can I make it available during indexing?

Never mind, found it. I created #65 and implemented it. It will be released on 1.7.0
